docs/guide/upgrade/migrate_v1_v2.md

```diff
@@ -15,17 +15,12 @@ It supports existing AWS resources provisioned by AWSALBIngressController(>=v1.1
 
 1. The AWS LoadBalancer resource created for your Ingress will be preserved.
     
-2. If a numeric TargetPort is used in your service, the AWS TargetGroups created for your Ingress will be re-created.
-
-    !!!warning "downtimes"
-        This would cause downtimes to your service during targets registration into new TargetGroups created.
-    
+2. The AWSALBIngressController always used `1` as TargetGroup's port. The AWSLoadBalancerController will use 
+     * the actual numeric `targetPort` of the corresponding Kubernetes `Service` as TargetGroup's port if a numeric `targetPort` is used.
+     * `1` as TargetGroup's port if a lexical `targetPort` is used.
+   
     !!!tip "details"
-        * The AWSALBIngressController always used `1` as TargetGroup's port.
-        * The AWSLoadBalancerController will use 
-            * the actual numeric TargetPort as TargetGroup's port if a numeric TargetPort used.
-            * `1` as TargetGroup's port if a lexical TargetPort used.
-        * The AWSLoadBalancerController will automatically create new TargetGroups and cleanup old TargetGroups if any.
+        Existing TargetGroups will be preserved; if a numerical `targetPort` is used, the TargetGroup's port will not be updated and stay `1`.
```

Comment:
if numerical `targetPort` is used, then the port will be updated

Reply:
This was a bit confusing, I got what you were trying to say.

Reply:
So should I remove the whole second point in that list?

Reply:
@devkid i think we should complete remove it. 
better to mention somewhere in the doc that this doc applies to v2.0.1.

Reply:
I prefer removing the second point completely.

Reply:
Second point is removed, version in the paragraph before the first point is updated to 2.0.1.

Reply:
Thanks.